### ISSUE TITLE
Feature: Keep query parameters

### ISSUE THREAD

**Issue by joehi** [inspiredminds/contao-sibling-navigation#4]
It would be nice to have in option to keep certain query parameters.

I have something in mind like at the package [changelanguage](https://github.com/terminal42/contao-changelanguage) where I have a field **Keep query parameters**.

The description could be: "Enter a comma separated list of query parameters that should be kept when linking."

An usecase would be to control with a query parameter whether to show or not show additional information like a map on the reader-page. I would like to reference for such a scenario to a [discussion](https://community.contao.org/de/showthread.php?74709-Inhaltselement-%C3%BCber-GET-Parameter-ein-ausblenden&p=501810&viewfull=1#post501810) in the Contao-forum.

**Comment by fritzmg** [comment]
Hm, well you could also add the necessary parameters in your own template.

**Comment by joehi** [comment]
Thank you for the hint. I will try it that way.

**Comment by joehi** [comment]
I created my own mod_sibling_navigation__modified template but I cannot select it from the "Custom module template" dialog. The selection stays empty. How can I select my own template?

**Comment by fritzmg** [comment]
Hm... try with `mod_sibling_navigation_events_modified.html5` for example.

**Comment by joehi** [comment]
Thank you. Now I could select it. I did not expect that I would have to add something to the official name to make it show up in the dialog.

**Comment by fritzmg** [comment]
Yeah it's a special case. It's two different modules - but only one template. To select a custom module template in Contao 4.4+, you need to name the template according to the module name.
